packages/cli/lib/auth.mjs

```diff
@@ -11,7 +11,7 @@ import url from 'node:url';
 import { logger } from './utils.mjs';
 
 const BASE_URL = process.env.BASE_URL || process.env.PLATFORM_URL;
-const CLIENT_ID = process.env.CLIENT_ID || process.env.API_USER || 'flow-cli';
+const CLIENT_ID = 'flow-cli';
```

Comment:
nee da kannst du es nicht ändern. wenn dann nur in der login function
service account login muss weiterhin funktionieren